swift/ql/lib/codeql/swift/elements/decl/VarDecl.qll

```diff
@@ -9,5 +9,5 @@ class VarDecl extends Generated::VarDecl {
 }
 
 class FieldDecl extends VarDecl {
```

Comment:
Maybe this should be PropertyDecl, to keep it consistent with the language's preferred name.